### PR TITLE
Updated compose to stable 1.0.0 version

### PR DESCRIPTION
**Pull request by Morfly**
Updated compose to stable 1.0.0 version.

Added workaround for 'kotlinx-coroutines-core-jvm' artifact with adding missing 'sun.misc' classes.
(as per this solution: [bazel_issue_13553](https://github.com/Bencodes/bazel_issue_13553/blob/bazel-issue-13553-desugar-workaround/WORKSPACE))

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -24,27 +25,39 @@ load("@rules_jvm_external//:defs.bzl", "maven_install")
...
+        "androidx.compose.material:material:{}".format(_COMPOSE_VERSION),
+        "androidx.compose.ui:ui:{}".format(_COMPOSE_VERSION),
+        "androidx.compose.ui:ui-tooling:{}".format(_COMPOSE_VERSION),
+        "androidx.compose.compiler:compiler:{}".format(_COMPOSE_VERSION),
+        "androidx.compose.runtime:runtime:{}".format(_COMPOSE_VERSION),
     ],
     override_targets = {
         "org.jetbrains.kotlin:kotlin-stdlib": "@com_github_jetbrains_kotlin//:kotlin-stdlib",
         "org.jetbrains.kotlin:kotlin-stdlib-jdk7": "@com_github_jetbrains_kotlin//:kotlin-stdlib-jdk7",
         "org.jetbrains.kotlin:kotlin-stdlib-jdk8": "@com_github_jetbrains_kotlin//:kotlin-stdlib-jdk8",
```
Comment (Morfly) [inline WORKSPACE]: I wonder, what was the purpose of adding overrides for kotlin-stdlib artifacts?
The project can be built without them as well

File: BUILD [hunk 2]
```diff
@@ -40,3 +47,15 @@ kt_compiler_plugin(
         "@maven//:androidx_compose_compiler_compiler",
     ],
 )
+
+# Add missing 'sun.misc' files to coroutines artifact
+# Used in 'override_targets' by referencing @//:kotlinx_coroutines_core_jvm
+kt_jvm_import(
+    name = "kotlinx_coroutines_core_jvm",
+    jars = ["@maven_secondary//:v1/https/repo1.maven.org/maven2/org/jetbrains/kotlinx/kotlinx-coroutines-core-jvm/1.5.1/kotlinx-coroutines-core-jvm-1.5.1.jar"],
```
Comment (Bencodes) [inline BUILD:55]: You also need to pass the source jar otherwise it can break the IntelliJ Bazel Plug-in aspect.

File: WORKSPACE (hunk 1 above)
Comment (Bencodes) [reply WORKSPACE]: You can remove the overrides

Comment (Morfly) [reply WORKSPACE]: Removed the overrides

File: BUILD (hunk 2 above)
Comment (Morfly) [reply BUILD:55]: Added `srcjar`

**Review by Bencodes — APPROVED** [review]

**Comment by Bencodes** [comment]
Looks good. Thanks for the pull request!

**Pull request merged**